src/sardana/macroserver/macros/examples/parameters.py

```diff
@@ -107,7 +158,12 @@ def run(self, *args, **kwargs):
         pass
     
 class pt7(Macro):
-    """Macro with a list of pair Motor,Float"""
+    """Macro with a list of pair Motor,Float.
+    Usages, ex.:
+    pt7 [[mot1 1][mot2 3]]
+    pt7 [mot1 1 mot2 3]
```

Comment:
After implementing changes agreed in the last #405 comments this should not be accepted. Please remove if you agree.